pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[tool.poetry]
+name = "forecast-solar"
+version = "0.0.0"
+description = "Asynchronous Python client for getting forecast solar information"
+authors = ["Klaas Schoute <hello@student-techlife.com>"]
+maintainers = ["Klaas Schoute <hello@student-techlife.com>"]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/home-assistant-libs/forecast_solar"
+repository = "https://github.com/home-assistant-libs/forecast_solar"
+documentation = "https://github.com/home-assistant-libs/forecast_solar"
+keywords = ["forecast", "solar", "power", "energy", "api", "async", "client"]
+classifiers = [
+    "Framework :: AsyncIO",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+packages = [
+    { include = "forecast_solar", from = "src" },
+]
+
+[tool.poetry.dependencies]
+aiohttp = ">=3.0.0"
+aiodns = ">=3.0.0"
+python = "^3.10"
```

Comment:
Will you bump the minimum Python version to 3.11 in a separate PR and label it as breaking?

Reply:
In a separate PR after or before the merge of this PR? This Poetry PR already brings the minimum version of Python 3.7 to 3.10, perhaps we should immediately make it 3.11 and label this PR as a breaking change?